find/scripts/extract_download_logs.py

```diff
@@ -21,8 +21,6 @@
 from notifications_python_client import prepare_upload
 from notifications_python_client.notifications import NotificationsAPIClient
 
-from config import Config
-
```

Comment:
Can we revert changes to this file? It should work now by changing the run command only.

Reply:
Zoltan and I were chatting about this.

It doesn't work because if you try to load the `Config` object you need to inject a bunch of other environment variables, which the download-report job doesn't have access to.

It feels redundant to add them all when they're unused. The only thing that is needed is the account-store API host, and this is hardcoded directly in the service manifest.

So it feels easier to just hardcode it here too. I appreciate it's a little bit clunky - but it's that, or update the download-report job manifest to include a bunch of env vars/secrets that it has no use for (incl. the DB secret, which feels less than ideal), or rework the config object further to consider more things optional (which I don't think is good).

This feels the least-worst thing to get back on track easily.

Reply:
As discussed with @gidsg , Config is no longer needed, ACCOUNT_STORE_API_HOST has been added to manifest.yml, and gets retrieved directly in the script.